starter-code/index.html

```diff
@@ -9,6 +10,30 @@
   <script type="text/javascript" src="./js/index.js"></script>
   <title>Ironhack cart</title>
 </head>
+
 <body>
+  <h1>Ironhack Merchandising Shop</h1>
+  <div class="wraper">
+    <div>
+      <span>Ironhack course</span>
+    </div>
+    <div>
+      <span class="item-price">25.00</span>
+    </div>
+    <div>
+      <label class="quantity">QTY</label>
+      <input type="number">
+    </div>
+    <div>
+      <span id="total">$00.00</span>
+    </div>
+    <div>
+      <button class="btn-delete btn">Delete</button>
+    </div>
+  </div>
+  <div id="total-container">
+    <button onclick="getTotalPrice()" id="calc-prices-button" class="btn-success btn">Calculate Prices</button>
```

Comment:
The use of onclick is cheating :stuck_out_tongue_closed_eyes: In your JS there is code to help you do this without adding the onClick:
```
var calculatePriceButton = document.getElementById('calc-prices-button');
calculatePriceButton.onclick = getTotalPrice;
```